partition/roles/dhcp/tasks/main.yaml

```diff
@@ -4,26 +4,60 @@
     fail_msg: "not all mandatory variables given, check role documentation"
     quiet: yes
     that:
-      - dhcp_net is defined
-      - dhcp_netmask is not none
-      - dhcp_server_ip is not none
-      - dhcp_range_min is not none
-      - dhcp_range_max is not none
+      - item.network is defined
+      - item.netmask is not none
+      - item.range is not none
+      - item.range.begin is not none
+      - item.range.end is not none
+  loop: "{{ dhcp_subnets }}"
+
+- name: Get stat for SONiC's version file
```

Comment:
The role should not be aware of operating systems. Looks like the only purpose is to add the `force_apt_get: yes` parameter? Why not just pass it explicitly as a role parameter?